app/src/lib/git/status.ts

```diff
@@ -116,6 +144,11 @@ export async function getStatus(
   for (const entry of parsePorcelainStatus(stdout)) {
     if (entry.kind === 'entry') {
       const status = mapStatus(entry.statusCode)
+      const hasConflictMarkers = await entryHasConflictMarkers(
```

Comment:
Awaiting in a loop like this means we can't take advantage of concurrency here. Is that the goal in this case? I know we've had issues in the past with Git processes being spawned all over the place, but is there another strategy to make sure this function doesn't take a long time to complete (imagine having 10k files that need to be processed)?

Reply:
mmm good call! i'll fix that 🔧

Reply:
We may want to keep it this way. I only brought it up to make sure. It's never good to have 10K git processes spawned, which doing this concurrently could cause, so if you were to remove await, we'd want to limit the amount of Git processes that are spawned by employing some sort of rate limiting strategy.

cc @desktop/eng for feels.